.github/workflows/attack-coverage-update.yml

```diff
@@ -0,0 +1,68 @@
+name: attack-coverage-update
+on:
+  workflow_dispatch:
+      inputs:
+        update-coverage:
+          description: 'Update the docs/ATT&CK-coverage.md file'
+          required: true
+          default: '--update-coverage'
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout detection-rules
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip cache purge
+          pip install .[dev]
```

Comment:
Just double checking, might you need lib/kibana and lib/kql as well or perhaps not?

Reply:
I think we don't need them, we have mimed the workflow of .github/workflows/pythonpackage.yml where the artifacts are built and naviagtor gist is updated, which does not use these libs

Reply:
:thinking: That seems strange to me given that the `build-release` command is run as part of this workflow I think does require it.

Reply:
It is being installed as part of pip. Refer - https://github.com/elastic/detection-rules/blob/main/pyproject.toml#L41

Also verified from our workflow Job. 

<img width="936" alt="image" src="https://github.com/user-attachments/assets/48682fa0-207a-4d45-94e3-43cdf4c43fab">

We do not need an explicit install

Reply:
Ah its included in `.[dev]` now, so possibly no longer needed in the Makefile. 

<details><summary>Details</summary>
<p>

```bash
detection-rules on  main [$!?] is  v0.1.0 via  v3.12.6 (test_venv) on  eric.forte 
❯ pip install .[dev]
Looking in indexes: https://pypi.org/simple, https://eric.forte%40elastic.co:****@artifactory.elastic.dev/artifactory/api/pypi/pypi-endgame/simple
Processing /home/forteea1/Code/clean_mains/detection-rules
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting detection-rules-kql@ git+https://github.com/elastic/detection-rules.git#subdirectory=lib/kql (from detection_rules==0.1.0)
  Cloning https://github.com/elastic/detection-rules.git to /tmp/pip-install-dcquu3e4/detection-rules-kql_d349e29b7d7c4178a81dd9eaae6635b8
  Running command git clone --filter=blob:none --quiet https://github.com/elastic/detection-rules.git /tmp/pip-install-dcquu3e4/detection-rules-kql_d349e29b7d7c4178a81dd9eaae6635b8
  Resolved https://github.com/elastic/detection-rules.git to commit 5e0fb4a63eb7a93f08b691a73ce3aa97a975fde6
  Running command git submodule update --init --recursive -q
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting detection-rules-kibana@ git+https://github.com/elastic/detection-rules.git#subdirectory=lib/kibana (from detection_rules==0.1.0)
  Cloning https://github.com/elastic/detection-rules.git to /tmp/pip-install-dcquu3e4/detection-rules-kibana_6ca2981e007a4fd0aa8a1c0cb209f3f9
  Running command git clone --filter=blob:none --quiet https://github.com/elastic/detection-rules.git /tmp/pip-install-dcquu3e4/detection-rules-kibana_6ca2981e007a4fd0aa8a1c0cb209f3f9
```

</p>
</details>